test/e2e/kubelet.go

```diff
@@ -228,4 +404,87 @@ var _ = framework.KubeDescribe("kubelet", func() {
 			})
 		}
 	})
+
+	// Delete nfs server pod after another pods accesses the mounted nfs volume.
+	framework.KubeDescribe("host cleanup with volume mounts [HostCleanup]", func() {
```

Comment:
@gnufied It may turn out to be a good idea to create a new files for these tests; however, atm I am ok with keeping them in _kubelet.go_. Reasons: 1) using nfs is somewhat incidental since the main purpose of the test is to verify that the kubelet cleans up volume mounts and the pod's uid dir. I don't foresee needing this type of test for every in-tree storage plugin. 2) some of the existing tests in _kubelet.go_ are disruptive and so it seems that mine fits ok there, at least for now.